src/devices_models/device_constructors/branch_constructor.jl

```diff
@@ -253,12 +259,17 @@ function construct_device!(
     sys::PSY.System,
     ::ModelConstructStage,
     device_model::DeviceModel{T, StaticBranchBounds},
-    network_model::NetworkModel{S},
-) where {T <: PSY.ACBranch, S <: StandardPTDFModel}
+    network_model::NetworkModel{<:StandardPTDFModel},
```

Comment:
This network type is concrete 
```suggestion
    network_model::NetworkModel{StandardPTDFModel},
```